### DISCUSSION TITLE
Welcome to redir Discussions!

### DISCUSSION THREAD

**Discussion by troglobit** [troglobit/redir#34] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by boussou** [comment]
Hi there, I am using it to temporarily access to docker infra from the outside.
I like it because it will not stay open if I forget the console open & close the laptop for example.
simple & effective
